src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/RazorConfiguration.cs

```diff
@@ -16,7 +16,8 @@ public sealed record class RazorConfiguration(
     bool SuppressAddComponentParameter = false,
     LanguageServerFlags? LanguageServerFlags = null,
     bool UseRoslynTokenizer = false,
-    LanguageVersion CSharpLanguageVersion = LanguageVersion.Default)
+    LanguageVersion CSharpLanguageVersion = LanguageVersion.Default,
+    string? RootNamespace = null)
```

Comment:
Looks like this isn't referenced anywhere from the compiler layer; why does it need to be in the compiler's RazorConfiguration?

Reply:
`RazorConfiguration` serves double duty right now, as the thing that the compiler reads _some_ things from, and the type that stores all of the compiler configuration knobs that are serialized between the IDE host and the LSP server. There has been much discussion about removing the `configure` lambdas for configuring the compiler, and moving to fully utilise the `RazorConfiguration`, so this is something of a hedge towards that.